Packs/IdentityRecordedFuture/pack_metadata.json

```diff
@@ -20,12 +20,56 @@
         "identity",
         "identities"
     ],
-    "dependencies": {},
+    "dependencies": {
+        "PAN-OS": {
+            "mandatory": false,
+            "display_name": "PAN-OS by Palo Alto Networks"
+        },
+        "Okta": {
+            "mandatory": false,
+            "display_name": "Okta"
+        },
+        "Malware": {
+            "mandatory": false,
+            "display_name": "Malware Core"
+        },
+        "Active_Directory_Query": {
+            "mandatory": false,
+            "display_name": "Active Directory Query"
+        },
+        "CoreAlertFields": {
+            "mandatory": true,
```

Comment:
this is super weird - why does this pack rely on something for XSIAM content as mandatory?
Asking since for XSOAR this pack won't be available at all...

Reply:
I don't think I understand the question.
As well as I'm not sure how to address/answer it.

Those dependencies were generated by demisto-sdk.